client/.env.development

```diff
@@ -1,3 +1,3 @@
-HOST=local.reversim.com
```

Comment:
The reason it is `local.reversim.com` is for google auth to work when running locally. 
Add to your hosts file:

```
$  cat /etc/hosts
...
127.0.0.1 local.reversim.com.
...
```

Not required for now, but you'll need it later